### DISCUSSION TITLE
nextjs 13 data not making it to the backend?

### DISCUSSION THREAD

**Discussion by GollyJer** [pmndrs/jotai#1834] [Q&A]
I don't understand why or even what is happening. Hoping someone can help.
I am loading my app from a query and after the query returns I'm setting the data into an atom.

But, after it happens the page doesn't load and the console looks like this.
<img width="488" alt="macos_screenshot_2023-03-11 19 00 58_dg3cz1" src="https://user-images.githubusercontent.com/689204/224521961-dfa49430-acf7-4c0f-bd32-b6e81088e286.png">
When the items are set to 1000 is when the query returned with data and is sent to the atom. At that point it's 1000 items, but the `devtools_backend` still shows 0 items.

Thanks for any pointers in understanding this. 👍

**Comment by dai-shi** [comment]
Can you identify if it's nextjs specific behavior, or otherwise can you reproduce it with https://csb.jotai.org ?
